README.Rmd

```diff
@@ -11,68 +11,67 @@ knitr::opts_chunk$set(
-
-**Rename T-cell receptor genes between 10X, Adaptive, and IMGT formats**
+# Convert TCR gene names
 
-TCRconvertR takes T-cell receptor (TCR) data containing V, D, J, and/or C genes from 10X, Adaptive, or other sequencing platforms and renames them from any of these formats to any other one:
+`TCRconvertR` converts V, D, J, and/or C gene names between the 10X Genomics, Adaptive Biotechnologies, and IMGT nomenclatures. It supports alpha-beta and gamma-delta T cell receptors (TCRs) for human, mouse, and rhesus macaque. Users can also define custom species, see: `vignette("custom-species")`. A [Python version](https://github.com/seshadrilab/tcrconvert) with command-line support is also available.
 
-* **10X**: TRAV1-2
-* **Adaptive**: TCRAV01-02*01
-* **IMGT**: TRAV1-2*01
+## Background
 
-TCRconvertR works with human, mouse, and rhesus macaque data out-of-the-box, but users can also add their own species.
+TCR annotation tools use different gene naming conventions, making cross-dataset searches difficult (e.g., identifying 10X-annotated TCRs in Adaptive data). Manual conversion is complex and error-prone due to inconsistencies in naming rules.
 
-TCRconvertR helps researchers unify TCR datasets by converting them to a standard naming convention. It is fast, reliable, and prevents errors from manual conversions. Unlike other tools that require custom objects, TCRconvertR works directly with DataFrames and CSV/TSV files.
+`TCRconvertR` automates this process efficiently and accurately. Our approach is based on analyzing multiple 10X and Adaptive data sets to capture their naming variations.
 
-# Installation
+## Installation
 
-You can install TCRconvertR from [GitHub](https://github.com/) with:
+Install from GitHub:
 
-``` r
+```r
 # install.packages("pak")
 pak::pak("seshadrilab/tcrconvertr")
 ```
 
-# Usage
+## Usage
 
-**Load some 10X data**
+#### 1. Load TCRs into a data frame
 
-```{r example}
-library(TCRconvertR)
+Examples of files you may want to load:
 
-tcr_file <- "/home/emmabishop/workspace/tcrconvertr/inst/extdata/examples/tenx.csv"
+- **10X**: `filtered_contig_annotations.csv`
+- **Adaptive**: `Sample_TCRB.tsv`
+- **IMGT**: Output from `MiXCR` or other tools
+
+```{r}
+library(TCRconvertR)
 
-tcrs <- read.csv(tcr_file)
+tcr_file <- get_example_path("tenx.csv") # Using built-in example file
+tcrs <- read.csv(tcr_file)[c("barcode", "v_gene", "j_gene", "cdr3")]
 tcrs
 ```
 
-**Convert gene names from the 10X format to the Adaptive format**
+####  2. Convert
 
 ```{r}
 new_tcrs <- convert_gene(tcrs, frm = "tenx", to = "adaptive")
 new_tcrs
 ```
 
-# Contributing
+## Contributing
 
-I welcome feedback! If you would like to resolve an issue or add improvements please submit a pull request.
+Feedback is welcome! To contribute, submit a pull request.
 
-# Issues
+## Issues
 
-If you run into problems or need help running TCRconvertR please file an issue on GitHub.
+For problems or questions, file an issue on GitHub.
```

Comment:
put a link behind `file an issue on GitHub` to https://github.com/seshadrilab/tcrconvertr/issues/new

Reply:
Fixed in [fb99828](https://github.com/seshadrilab/tcrconvertr/pull/14/commits/fb9982891f1de8a5ae42024d6d4633bc87b19e3a)